AzureFirewall/AzureFirewallNetworkRule.kql

```diff
@@ -0,0 +1,32 @@
+let AzureFirewallNetworkRulesTcp = AzureDiagnostics

```

Comment:
This is a good example of where union (which is not recommended if not needed) can be easily avoided.

Reply:
Yes agreed. Like I've stated above, I think I managed to improve the parsers now by using a combination of `where` and `parse-where` before performing the `union`.